cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/batch/distributed/DistributedMapReduceTaskContextProvider.java

```diff
@@ -126,31 +105,6 @@ protected void shutDown() throws Exception {
   }
 
   private static Injector createInjector(CConfiguration cConf, Configuration hConf) {
-    return Guice.createInjector(
```

Comment:
What do you mean if its ok to "refactor them as..." ?
Whether that will work? Or whether certain bindings are unnecessary?